nemo/collections/nlp/modules/common/megatron/token_level_encoder_decoder.py

```diff
@@ -290,15 +290,15 @@ def forward(
         dec_attn_mask,
         token_type_ids=None,
         labels=None,
-        enc_hidden_states=None,
-        enc_output_mask=None,
-        output_enc_hidden_only=False,
+        enc_output=None,
+        enc_output_attn_mask=None,
         enc_input=None,
+        output_enc_hidden_only=False,
     ):
         """
         Return value is per token / per dimension (i.e., non collapsed loss value)
         """
-        if enc_input is None:
+        if (enc_input is None) and (enc_output is None):
```

Comment:
what about the case enc_input is None but enc_output is not None? we are not going to compute the enc_input ?
why not just do `if enc_input is None`

Reply:
Fixed.